tests/test_wave_05.py

```diff
@@ -122,29 +144,28 @@ def test_delete_goal(client, one_goal):
 
     # Check that the goal was deleted
     response = client.get("/goals/1")
+    response_body = response.get_json()
     assert response.status_code == 404
-
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
+    assert response_body == {"error": "Item #1 cannot be found"}
+    assert Goal.query.all() == []
```

Comment:
This is also good test design, especially checking the DB and not just response.